app/controllers/guidance_groups_controller.rb

```diff
@@ -66,7 +66,7 @@ def admin_update_publish
     if @guidance_group.update_attributes(params[:guidance_group])
       redirect_to admin_index_guidance_path(params[:guidance_group]), notice: _('Guidance group was successfully updated.')
     else
-      render action: "edit"
+      redirect_to admin_index_guidance_path(@guidance_group), notice: generate_error_notice(@guidance_group)
```

Comment:
We should adopt this as general pattern. Much better than what we've been working with. It might be worth generalising the 'Unable to save your changes' message e.g under plan creation we could do the same but might want 'Unable to Create Plan'. Add as a param maybe?